src/v2/Apps/Conversation/ConversationApp.tsx

```diff
@@ -47,7 +47,12 @@ export const ConversationApp: React.FC<ConversationAppProps> = props => {
   }, [])
 
   useEffect(() => {
-    if (isEnabled && width > breakpoints.xs && conversation && router) {
+    if (
+      isEnabled &&
+      width > parseInt(breakpoints.xs, 10) &&
```

Comment:
Do you think it would be worth doing this conversion directly from the exposed breakpoints in palette?

Reply:
It would be nice to keep things consistent, but its also kinda annoying to have to do parseInt... I'm kinda divided about it, but leaning towards consistency.